halo2-snark-aggregator-api/src/tests/systems/halo2/add_mul_test/verify_single.rs

```diff
@@ -53,6 +53,7 @@ pub fn test_verify_single_proof_in_chip<
         Error = halo2_proofs::plonk::Error,
     >,
 {
+    println!("test_verify_single_proof_in_chip");
```

Comment:
Useless println should be removed later

Reply:
fixed with https://github.com/scroll-tech/halo2-snark-aggregator/pull/35/commits/fd933c78f3dede595eaff61c9cc0505b2a94afc2